src/App.tsx

```diff
@@ -1,11 +1,14 @@
 import React from 'react';
 import './App.css';
 import Header from './Components/Header';
+import Footer from './Components/Footer';
 
 function App() {
   return (
     <div className="App">
       <Header/>
+      {/* <div className="testdiv">Testing...</div> */}
```

Comment:
many tech leads don't allow any commented out code let alone commented out code that is obviously never going to be used.  I think you should get in the habit at ey to keep code clean

Reply:
I agree I merged so that Charmaine, can have her code updated to the latest, she is working in refactoring the navigation with routing.